viewing_party/party.py

```diff
@@ -1,23 +1,173 @@
+
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    
+    if title and genre and rating:
+        movie_dict = {}
+        movie_dict["title"] = title
+        movie_dict["genre"] = genre
+        movie_dict["rating"] = rating
+    else:
+        return None
+    
+    return movie_dict
+
+
+def add_to_watched(user_data, movie):
+
+    user_data["watched"].append(movie)
+
+    return user_data
+
+
+def add_to_watchlist(user_data, movie):
+
+    user_data["watchlist"].append(movie)
+
+    return user_data
+
+
+def watch_movie(user_data, title):
+
+#loop through the dictionaries in the list user_data["watchlist"]

```

Comment:
I would consider rewriting these comments to be a more spoken-language description, and break them up so they are placed close to the code that performs the action they describe. This is great pseudocode, but as comments for a final product, they are close enough to real code that they don't let a reader rapidly understand what to expect the code below to perform.

Reply:
Noted! This is actually pseudocode I forgot to delete. I was almost afraid to delete it because I had been struggling with this function and the pseudocode had helped unblock me. I will make a note to remember to delete pseudocode or turn it into a spoken-language description.